Quran/SQLiteSearchableAyahTextPersistence.swift

```diff
@@ -39,22 +69,33 @@ class SQLiteSearchableAyahTextPersistence: ReadonlySQLitePersistence, Searchable
     private func search(for term: String, connection: Connection) throws -> [SearchResult] {
         CLog("Search for:", term)
         let searchTerm = cleanup(term: term)
+        let arabicSearchTerm = searchTerm.replacingOccurrences(of: regex,
+                                                               with: "_",
+                                                               options: .regularExpression)
+        let pattern = generateRegexPattern(term: term)
         let query = Database.table
-            .select(Database.Columns.snippet, Database.Columns.sura, Database.Columns.ayah)
-            .filter(Database.Columns.text.match("\(searchTerm)"))
-            .limit(300)
```

Comment:
note - i intentionally dropped the `limit` because this opens up the search a lot wider, even though a lot fewer results come in.